artemis-server/src/main/java/org/apache/activemq/artemis/core/server/ActiveMQServerLogger.java

```diff
@@ -530,11 +529,6 @@ void slowConsumerDetected(String sessionID,
    @Message(id = 222023, value = "problem cleaning page address {0}", format = Message.Format.MESSAGE_FORMAT)
    void problemCleaningPageAddress(@Cause Exception e, SimpleString address);
 
-   @LogMessage(level = Logger.Level.WARN)
-   @Message(id = 222024, value = "Could not complete operations on IO context {0}",
```

Comment:
this just removes the log line we are seeing, isnt that now just masking the problem?

Reply:
No... we are using asynchronously now.. before it was blocking.. no need to the handler.

Reply:
On another fix there was a loop removed here.. the thread is exausting the pool and causing the log.warn.


notice I'm also fixing the context used on cleanup.

Reply:
just deployed to real server in testing env, seeing if issue of flow stopping is resolved. will let you know